lib/convert.js

```diff
@@ -17,22 +17,46 @@ var sdk = require('postman-collection'),
     generatedStore: {},
 
     /**
-     * Adds items from the trie as folders (or leaf requests) into the collection object
-     * @param {object} specWrapper - specWrapper.tree is trie generated from util
+     * Adds items from the trie/tags as folders (or leaf requests) into the collection object
+     * @param {object} specWrapper
+     * @param {object} specWrapper.tree - the trie generated from util
+     * @param {object} specwrapper.tags - the tag folder object generated from util
      * @returns {void}
      */
     generateCollection: function(specWrapper) {
-      var folderTree = specWrapper.tree, // this is the trie we generate (as a scaffold to the collection)
-        openapi = specWrapper.spec, // this is the JSON-version of the openAPI spec
-        child;
-
-      for (child in folderTree.root.children) {
-        // A Postman request or folder is added if atleast one request is present in that sub-child's tree
-        // requestCount is a property added to each node (folder/request) while constructing the trie
-        if (folderTree.root.children.hasOwnProperty(child) && folderTree.root.children[child].requestCount > 0) {
-          this.generatedStore.collection.items.add(
-            util.convertChildToItemGroup(openapi, folderTree.root.children[child])
-          );
+      var tagsFolderObject, folderTree, openapi, child, tag;
+      openapi = specWrapper.spec;
+      if (specWrapper.tags) {
+        tagsFolderObject = specWrapper.tags;
+        for (tag in tagsFolderObject) {
+          if (tagsFolderObject.hasOwnProperty(tag)) {
+            if (tag === 'no-tag') {
```

Comment:
what are the permissible values of this?
On first glance, this looks like it prevents tags with the name `no-tag`. We should use `null`, or use a separate property